Source/Csla/Core/MobileBindingList.cs

```diff
@@ -24,7 +27,11 @@ namespace Csla.Core
   [System.Diagnostics.DebuggerStepThrough]
 #endif
   [Serializable]
-  public class MobileBindingList<T> : BindingList<T>, IMobileList
+  public class MobileBindingList<
+#if NET8_0_OR_GREATER
+    [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]
```

Comment:
This requirement comes from `System.ComponentModel.BindingList<T>`.just have to propagate it.